test/extended/util/client.go

```diff
@@ -1165,3 +1149,13 @@ func (c *CLI) assembleArgs() []string {
 	finalArgs = append(finalArgs, c.flags...)
 	return finalArgs
 }
+
+func (c *CLI) isKCPCommand() bool {
+	kcpCommands := []string{"kcp", "ws", "workspaces"}
+	for _, cmd := range kcpCommands {
+		if cmd == c.command {
+			return true
+		}
+	}
+	return false
```

Comment:
```suggestion
	return StrSliceContains(kcpCommands, c.command)
```
Seems we could reuse the function.

Reply:
Hi @zxiao-redhat In addition, could you please help run all the smoke cases to make sure we don't miss other things need to adapt? Thanks!